src/transformation/visitors/return.ts

```diff
@@ -59,6 +60,11 @@ export const transformReturnStatement: FunctionVisitor<ts.ReturnStatement> = (st
         insideTryCatch = insideTryCatch || scope.type === ScopeType.Try || scope.type === ScopeType.Catch;
     }
 
+    const result = transformMultiReturnStatement(context, statement);
```

Comment:
Also here would be nice if you could check if this is a multi return before transforming. This is to prevent getting huge complexities randomly (happened before), because we are doing more work than we really need to.